.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: /
+  schedule:
+    interval: daily
```

Comment:
@JAVGan Very nice idea to sue dependabot
If we want to have daily; than it should be just for CVE fixes 

Others should be done weekly
And IIRC it can even generate group PR instead of separate PR for each project(dependency).


other PRS sh